upgrade/upgrade-scripts/verify

```diff
@@ -75,36 +80,70 @@ function cleanup() {
 	return "$rc"
 }
 
-trap cleanup EXIT
+#
+# Caller can specify container if they already have one. Java stack is a consumer of this feature
+# since it needs to know container name in two different points of stack. The 'verify' script needs
+# to report progress and returning container name along with progress report is disadvised.
+# We also want to keep 'verify' script to do all the verification for convenience reason.
+# By allowing caller to provide container name, we reuse verification logic rather than duplicating
+# them in here and in the Java stack.
+#
+function usage() {
+	echo "$(basename "$0"): $*" >&2
+
+	PREFIX_STRING="Usage: $(basename "$0")"
+
+	echo "$PREFIX_STRING [-c container-name] "
+
+	exit 2
+}
+
+while getopts ':c:' c; do
+	case "$c" in
+	c)
+		CONTAINER=$OPTARG
+		CALLER_PROVIDED_CONTAINER=true
+		;;
+	esac
+done
+shift $((OPTIND - 1))
```

Comment:
Unfortunately, `shellcheck` throws an error if `*)` case is missing.
However, it isn't possible for this script since rest of the arguments are implicitly treated as argument to `verify-jar`.
There is an [open conversation](https://github.com/delphix/appliance-build/pull/241/files#r262644047) on whether we want to remove `verify` script and call `verify-jar` and `verify-stack` directly.

Reply:
We decided `verify` script will be removed in favor of ["dry-run" functionality](https://github.com/delphix/appliance-build/pull/243/files) in `upgrade` script